prac_09/band.py

```diff
@@ -0,0 +1,22 @@
+"""Band class"""
+
+
+class Band:
+    def __init__(self, name=''):
+        self.name = name
+        self.musicians = []
+
+    def __repr__(self):
+        for musician in self.musicians:
+            return f"{self.name} {musician}"
+
+    def add(self, musician):
+        self.musicians.append(musician)
+
+    def play(self):
+        for musician in self.musicians:
+            if not musician.instruments:
```

Comment:
The musician class already handles this, you just need to through each member and use the `.play()` method.